src/Utils/CatIDToLabel.js

```diff
@@ -0,0 +1,18 @@
+import { isEmpty, isNil } from 'lodash';
+
+const CatIDToLabel = (data, catItems) => {
```

Comment:
basically, I think it's a common rule in JS world about naming, more or less equal to Google's https://google.github.io/styleguide/jsguide.html#naming:
* Class, interface, record, React component names are written in `UpperCamelCase`;
* Method names are written in `lowerCamelCase`;